infiny.js

```diff
@@ -1,12 +1,11 @@
-//To send 100 text messages of text "Blah" with a delay of 1 second after every 10 messages
-var input = document.querySelector('.block-compose .input');
-
-for(var i=0;i<10;i++){
-	setTimeout(function(){
-		for(var j = 0; j < 10; j++) {
-  			input.innerHTML = "Blah";
-    			input.dispatchEvent(new Event('input', {bubbles: true}));
-			var button = document.querySelector('button.compose-btn-send');
-			button.click();
-		}
-	},1000);}
+//To send 100 text messages of text "Blah" with a delay of 1 second
+var input = document.querySelector('.pluggable-input-body');
+var count = 0;
```

Comment:
Does the browser hang when you give a count >250. Can you please check it

Reply:
I can run it without any error or lag in chrome. Tried with 1000